OracleGoldenGate/21c/install-deployment.sh

```diff
@@ -44,6 +44,23 @@ function ogg_installer_setup() {
     unzip -q "${INSTALLER}" -d "/tmp/installer" || abort "Unzip operation failed for '${INSTALLER}'"
     chmod -R o=g-w             "/tmp/installer"
 }
+##
```

Comment:
This function violates all kinds of `shellcheck` rules. All shell scripts must pass `shellcheck` without warnings.